test/inductor/test_aot_inductor.py

```diff
@@ -104,7 +115,18 @@ def run_multiple(
         return list_output_tensors
 
 
-class AotInductorTests(TestCase):
+def check_model(
+    self: TestCase,
+    model,
+    example_inputs,
+):
+    expected = model(*example_inputs)
+    with config.patch("aot_inductor.abi_compatible", self.abi_compatible):
```

Comment:
I haven't turned on abi_compatible for every tests, and some of the fixes will come later.